logger.go

```diff
@@ -96,6 +96,13 @@ func (logger *Logger) Logf(level LogLevel, format string, args ...interface{}) {
 	}
 }
 
+func (logger *Logger) Write(p []byte) (n int, err error) {
+	logEntryMessage := NewLogEntryMessage(logger.scopes, InfoLevel, string(p))
```

Comment:
Should we use `logger.level` here instead of hardcoded `InfoLevel`?

Reply:
Yeah, wasn't sure about this one since `logger.level` is the level of the logger. I was thinking if we need to add something like:

```golang
if logger.IsLogLevelEnabled(InfoLevel) {
  return 0, nil
}
```

My thinking was that if you pass a logger as a Writer you want to log at info level there...

I also though about making a method like `AsWriter(level LogLevel)` that will return an `io.Writer`. What do you think?